src/Collections/Parameters.php

```diff
@@ -50,12 +50,13 @@ protected function doExport()
     /**
      * Searches whether a parameter with the given unique combination exists.
      */
-    public function has(string $name, string $in): bool
+    public function has(string $name, string $in = null): bool
     {
-        return isset($this->parameters[$name.'/'.$in]);
+        $id = $in ? $name.'/'.$in : $name;
+        return isset($this->parameters[$id]);
     }
 
-    public function get(string $name, string $in): Parameter
+    public function get(string $name, string $in = null): Parameter
```

Comment:
If I revert, then you could never find that parameter again. E.g:

```php
$operation->getParameters()->add(new Parameter(['$ref' => '#/parameters/foo']));
```
There is no way to check it with `has()` or to `get()` it. Ideally I would look at the target of the reference, then read the `name` and `in` but that is not possible.